argus/action_manager/windows.py

```diff
@@ -115,32 +116,76 @@ def get_installation_script(self):
         self.download_resource("windows/installCBinit.ps1",
                                r"C:\installCBinit.ps1")
 
+    def _execute_function(self, cmd, count=util.RETRY_COUNT,
```

Comment:
This can be named `_execute` like in other places https://github.com/cloudbase/cloudbase-init-ci/blob/master/argus/recipes/base.py#L51